### DISCUSSION TITLE
Version 0.4.1 Help!

### DISCUSSION THREAD

**Discussion by prs725** [niklasvh/html2canvas#2791] [Q&A]
We were using 0.5.0-beta4 version of html2canvas with Google Maps version 3.38. We recently are showing blank canvas that randomly started occurring. Trying out all the versions with Google Maps version 3.45 is not working. In the Issues, we found a lot of folks are using 0.4.1 as it works perfectly. We would like to use that however cannot find the integrity and tarball in the registry here: https://registry.npmjs.org/html2canvas 

Can someone please provide us the resolved (tarball) and integrity for version 0.4.1?

**Comment by prs725** [comment]
@niklasvh tagging you to get your attention, appreciate your help.

**Reply by AgentSmith0** [reply]
As far as I know, html2canvas version 0.4.1 isn't supported anymore, only the latest available version. So you would probably need to upgrade first.
